app/scripts/modules/kubernetes/src/v2/manifest/showManifestYaml.component.ts

```diff
@@ -0,0 +1,41 @@
+import { copy, IComponentOptions, IController, IScope, module } from 'angular';
+import { IModalService } from 'angular-ui-bootstrap';
+import { dump } from 'js-yaml';
+
+import { ServerGroupTemplates } from '@spinnaker/core';
+
+class KubernetesShowManifestYaml implements IController {
+  public manifest: any;
+  public text: string;
+  public linkName: string;
+  private title: string;
+
+  constructor(private $scope: IScope,
+              private $uibModal: IModalService) {
+    'ngInject';
+    this.text = dump(copy(this.manifest));
+    this.title = this.manifest.metadata.name;
+  }
+
+  public openYaml() {
+    this.$scope.userDataModalTitle = this.title;
```

Comment:
I would create a new scope rather than share `$scope` with this component. Inject `$rootScope` in the constructor, then 

```
const scope = $rootScope.$new();
scope.userDataModalTitle = this.title;
...
```

Reply:
Oh sweet - thanks